gapic-generator-java/src/test/java/com/google/api/generator/gapic/composer/grpc/goldens/DeprecatedServiceStubSettings.golden

```diff
@@ -102,6 +102,12 @@ public class DeprecatedServiceStubSettings extends StubSettings<DeprecatedServic
             "Transport not supported: %s", getTransportChannelProvider().getTransportName()));
   }
 
+  /** Returns the default service name. */
+  @Override
+  public String getServiceName() {
+    return "";
+  }
```

Comment:
Probably can exclude this from being generated if it's going to return `""`. The parent StubSettings already returns `""`